### DISCUSSION TITLE
How to get a project like this running again?

### DISCUSSION THREAD

**Discussion by JackySw** [Tichau/FileConverter#342] [General]
Hi,

This is about FileConverter specific and a bit about all Github projects in general. 

I use FileConverter for years and many do with me. But Tichau has stopped maintaining this project for a long time now. The last release was 2017. In the meantime, many new formats have emerged and requested. Slowly the program will become obsolete. Which is a shame since it is such a clean and powerful tool like no other. Not blaming Tichau, since he had put in hundreds of hours already! 

But the Issue list is growing and growing, still pull request are made, discussions fed, but the project is stuck for 6 years now. And I see this with many other projects.

How to revive such a project? I cannot do it myself do lack of knowledge of the programming language. Maybe learn it, specially for this project alone?

I always thought that Github is about working together to create something, but as I see now, if the author stops, the project dies.

Any thoughts on this? An idea how to turn this around?

**Comment by Tichau** [comment]
Hello,

I'd be happy to have someone that helps me to maintain this project. I've put a lot of time recently to go through a lot of technical debt in the project and make a release. I'll make some support and small bugfix in coming weeks but I will then probably stop working on it for a while.

The best way to help the project is to make direct pull request to it but I understand that this request a lot of investment. Some help can also be done maintaining healthy issues list and discussion, closing duplicates and categorizing them.

If you want to discuss more about this, don't hesitate.
